packages/mdc-select/mdc-select.scss

```diff
@@ -205,3 +225,8 @@
   cursor: default;
   pointer-events: none;
 }
+
+.mdc-select__menu .mdc-list .mdc-list-item--selected {
+  @include mdc-theme-prop(color, on-surface);
+  @include mdc-states;
```

Comment:
I'd suggest specifying `on-surface` here so it's clear this is intentionally aligning with the color above. (Took me a minute to realize on-surface is currently the default for states.)